dot/parachain/availability-store/availability_store.go

```diff
@@ -83,23 +205,33 @@ func (as *AvailabilityStore) loadMetaData(candidate common.Hash) (*CandidateMeta
 }
 
 // storeMetaData stores metadata in the availability store
-func (as *AvailabilityStore) storeMetaData(candidate common.Hash, meta CandidateMeta) error {
+func (as *AvailabilityStore) writeMeta(batch *AvailabilityStoreBatch, candidate parachaintypes.CandidateHash,
+	meta CandidateMeta) error {
 	dataBytes, err := json.Marshal(meta)
 	if err != nil {
 		return fmt.Errorf("marshalling meta for candidate: %w", err)
 	}
-	err = as.metaTable.Put(candidate[:], dataBytes)
+	err = batch.metaBatch.Put(candidate.Value[:], dataBytes)
 	if err != nil {
 		return fmt.Errorf("storing metadata for candidate %v: %w", candidate, err)
 	}
 	return nil
 }
 
+func (as *AvailabilityStore) deleteMeta(batch *AvailabilityStoreBatch, candidate parachaintypes.CandidateHash) error {
+	err := batch.metaBatch.Del(candidate.Value[:])
+	if err != nil {
+		return fmt.Errorf("deleting meta for candidate %v: %w", candidate, err)
+	}
+	return nil
+}
+
 // loadChunk loads a chunk from the availability store
-func (as *AvailabilityStore) loadChunk(candidate common.Hash, validatorIndex uint32) (*ErasureChunk, error) {
-	resultBytes, err := as.chunkTable.Get(append(candidate[:], uint32ToBytes(validatorIndex)...))
+func (as *AvailabilityStore) loadChunk(candidate parachaintypes.CandidateHash, validatorIndex uint32) (*ErasureChunk,
```

Comment:
```suggestion
func (as *AvailabilityStore) loadErasureChunk(candidate parachaintypes.CandidateHash, validatorIndex uint32) (*ErasureChunk,
```

Reply:
This name was used for consistency with names used by Parity in Polkadot codebase.